### DISCUSSION TITLE
Making a label bold in quasar QInput.

### DISCUSSION THREAD

**Discussion by harshalkumar-ishi** [pydantic/pydantic#7205] [Projects using Pydantic]
How can i make a label bold in Quasar QInput field.

I have used quasar typography classes "text-weight-bold"
and also used style.

**Comment by Viicos** [comment]
Wrong repo

**Reply by harshalkumar-ishi** [reply]
Which one is right then?

**Reply by Viicos** [reply]
https://github.com/quasarframework/quasar I guess

**Reply by harshalkumar-ishi** [reply]
Thank you.
